### ISSUE TITLE
Keyboard shortcut doesn't work on Mac OS X.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#107]
Originally reported on Google Code with ID 105

```
I have had two reports from users about this issue.

katherine.wai is using Chrome Beta (currently 8.0.552.210) on Mac OS X.

ScarfaceDeb did not mention their OS or Chrome version.

However, since it appears to work on GNU/Linux (stable and dev), and the number of
complaints have been small, I'm assuming this is a Mac OS X problem only.

I assume it's some bug in the Platform API. Or maybe Chrome for Mac is just buggy.
Luckily, I should have temporary access to a Mac OS X machine, so I may be able to
track this bug down...
```

Reported by `pcxunlimited` on 2010-11-27 16:50:53

**Comment by Pauan** [comment]
```
Okay, I tested this out, and it seemed to work okay. The ⌘ key didn't work, though,
oddly enough... Control did, though. The key was apparently detected okay, it just
didn't trigger the popup.
```

Reported by `pcxunlimited` on 2010-11-29 03:25:43

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-04-30 23:21:50
- **Labels added**: Component-arc2js
- **Labels removed**: Component-Platform

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 22:11:34
- **Labels removed**: Component-arc2js

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:04:23
- **Labels added**: Type-External
- **Labels removed**: Type-Bug

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:09:24
- **Status changed**: `Available`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:32:45
- **Labels added**: Type-Visible
- **Labels removed**: Type-External

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:36:10
- **Labels added**: Type-Confirmed
- **Labels removed**: Type-Visible

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:39:18
- **Labels removed**: Priority-Medium

**Comment by skodaikk** [comment]
Is the bug fixed? I am still not able to get the keyboard shortcut working on OSX 10.11.4 using Chrome 50.0.

**Comment by Pauan** [comment]
@skodaikk Are you using Tab Organizer from the Chrome Web Store, or are you using Tab Organizer from Github?

**Comment by skodaikk** [comment]
Hi,

I am using tab organizer from the chrome web store.

Thanks,
Sidharth

On Fri, May 6, 2016 at 5:23 AM, Pauan notifications@github.com wrote:

> @skodaikk https://github.com/skodaikk Are you using Tab Organizer from
> the Chrome Web Store, or are you using Tab Organizer from Github?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/Pauan/tab-organizer/issues/107#issuecomment-217425978

## 

KV Sidharth,
UC San Diego

**Comment by Pauan** [comment]
@skodaikk Unfortunately, the bug is not fixed with the version in the Chrome Web Store (version 4).

When version 5 is released, this bug should be fixed.

**Comment by skodaikk** [comment]
Sounds good!

Thanks,
Sidharth

On Mon, May 9, 2016 at 6:09 AM, Pauan notifications@github.com wrote:

> @skodaikk https://github.com/skodaikk Unfortunately, the bug is not
> fixed with the version in the Chrome Web Store (version 4).
> 
> When version 5 is released, this bug should be fixed.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/Pauan/tab-organizer/issues/107#issuecomment-217859587

## 

KV Sidharth,
UC San Diego
